src/_pytest/compat.py

```diff
@@ -67,8 +67,17 @@ def num_mock_patch_args(function):
     mock_modules = [sys.modules.get("mock"), sys.modules.get("unittest.mock")]
```

Comment:
its a 2 element list, use a variable, and use 2 is checks at the end - its ridiculus to introduce that kind of helper function programming here

Reply:
"ridiculous"